packages/public/server/__tests-e2e__/entities/article.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * This file is part of Serlo.org.
+ *
+ * Copyright (c) 2013-2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2013-2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/serlo.org for the canonical source repository
+ */
+import {
+  clickForNewPage,
+  getByItemType,
+  getByItemProp,
+  goto,
+  getBySelector,
+  login,
+  logout,
+  randomText,
+  getByPlaceholderText,
+  saveRevision,
+  addContent,
+  openDropdownMenu
+} from '../_utils'
+import { pages, notifications, elements } from '../_config'
+
+const articleItemType = 'http://schema.org/Article'
+
+test('view article', async () => {
+  const path = '/math/example-content/example-topic-1/example-article'
+  const title = 'Example article'
+  const content = 'Hello World! 42'
```

Comment:
The state of that article is wrong

Reply:
Might be because of the breaking changes in the plugin states we deployed recently. So you probably need to update your example content types in the dump